courses/fundamentals_of_ada/adv_270_subprogram_contracts.rst

```diff
@@ -144,9 +144,9 @@ Preconditions and Postconditions
 Examples
 ----------
 
-.. include:: examples/210_subprogram_contracts/preconditions_and_postconditions.rst
+.. include:: examples/adv_270_subprogram_contracts/preconditions_and_postconditions.rst
 
-:url:`https://learn.adacore.com/training_examples/fundamentals_of_ada/210_subprogram_contracts.html#preconditions-and-postconditions`
+:url:`https://learn.adacore.com/training_examples/fundamentals_of_ada/adv_270_subprogram_contracts.html#preconditions-and-postconditions`
```

Comment:
The  learn URLs don't work anymore

Reply:
That will be true until we can update the Learn site (which doesn't happen often).
As this is only for advanced, I think it's OK for now (but I'm going to comment them out until then)